dashboard/test/ui/features/plc/courseUnitNavigation.feature

```diff
@@ -0,0 +1,30 @@
+@dashboard_db_access
+Feature: Basic navigation for PLC stuff
+
+Background:
+  Given I am a teacher
+  And I am enrolled in a plc course
+  Given I am on "http://studio.code.org/course/csp-support"
```

Comment:
So I'd like to do this, but now is not the best time. AllThePlcThings would get associated with the CSP-Supports course because we haven't built course seeding yet.